test/index.spec.js

```diff
@@ -390,10 +390,32 @@ describe('daemons', () => {
         if (err) throw err
```

Comment:
see: https://github.com/ipfs/js-ipfsd-ctl/pull/155#issuecomment-326970190

Reply:
Here is an example: https://github.com/ipfs/npm-go-ipfs-dep/blob/master/src/index.js#L47-L60

Reply:
As it is a temporary skip, is it ok if I just `os.platform() === 'win32'` ?

https://nodejs.org/api/os.html#os_os_platform

Reply:
I don't get https://github.com/ipfs/npm-go-ipfs-dep/blob/master/src/index.js#L47-L60 sorry!! :( it also depends on another module (go-platform) and file https://github.com/ipfs/npm-go-ipfs-dep/blob/master/src/check-support.js